src/Dnn.PersonaBar.Library/Controllers/IModulesController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using DotNetNuke.Entities.Modules;
+using DotNetNuke.Entities.Portals;
+
+namespace Dnn.PersonaBar.Library.Controllers
+{
+    public interface IModulesController
```

Comment:
Earlier it was in Dnn.PersonaBar.Prompt.Components

Reply:
You mean this controller and PortalHelper will be removed from Dnn.AdminExperienceExtensions?

Reply:
Yes PortalHelper was added by me in commit of DNN-10709